setup/_metapackage/setup.py

```diff
@@ -26,6 +26,7 @@
         'odoo-addon-mgmtsystem_nonconformity_hr>=15.0dev,<15.1dev',
         'odoo-addon-mgmtsystem_nonconformity_mrp>=15.0dev,<15.1dev',
         'odoo-addon-mgmtsystem_nonconformity_product>=15.0dev,<15.1dev',
+        'odoo-addon-mgmtsystem_nonconformity_quality_control_oca>=15.0dev,<15.1dev',
```

Comment:
Same here. Weird. We shouldn't update this manually I think

Reply:
Probably I did something wrong. I did a rebase if this can fix things.